Dockerfile

```diff
@@ -30,78 +33,96 @@ LABEL org.label-schema.vcs-url="https://github.com/usdot-fhwa-stol/carma-platfor
 LABEL org.label-schema.vcs-ref=${VCS_REF}
 LABEL org.label-schema.build-date=${BUILD_DATE}
 
-ENV NVIDIA_VISIBLE_DEVICES \
-    ${NVIDIA_VISIBLE_DEVICES:-all}
-ENV NVIDIA_DRIVER_CAPABILITIES \
-    ${NVIDIA_DRIVER_CAPABILITIES:+$NVIDIA_DRIVER_CAPABILITIES,}graphics
+# Specify which platform GPUs are available inside the container
+ENV NVIDIA_VISIBLE_DEVICES ${NVIDIA_VISIBLE_DEVICES:-all}
+
+# Specify which driver libraries/binaries will be mounted inside the container
+ENV NVIDIA_DRIVER_CAPABILITIES ${NVIDIA_DRIVER_CAPABILITIES:+$NVIDIA_DRIVER_CAPABILITIES,}graphics
     
-RUN apt-get update && apt-get install -y lsb-release && apt-get clean ALL
+# Avoid interactive prompts during the building of this docker image
+ARG DEBIAN_FRONTEND="noninteractive"
 
-ENV ROS_DISTRO kinetic
-RUN sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'     \ 
-    && apt-key adv --keyserver 'hkp://keyserver.ubuntu.com:80' --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654     \ 
-    && apt-get update     \ 
-    && apt-get install ros-kinetic-desktop-full python-rosinstall -y     \ 
-    && rosdep init
+RUN apt-get update && apt-get install -y lsb-release && apt-get clean ALL
 
-RUN apt-get update && \
-        DEBIAN_FRONTEND=noninteractive apt-get install -y \
-        git \
-        ssh \
-        ros-kinetic-rosbridge-server \
-        sudo \
-        tmux \
-        vim \
-        nano \
-        less \
-        curl \
+# Install ROS Noetic
+ARG ROS_DISTRO=noetic
+RUN sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list' \ 
+    && apt-key adv --keyserver 'hkp://keyserver.ubuntu.com:80' --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654 \ 
```

Comment:
Can you confirm this is still a valid install after the ROS GPG key expiration issue that happened a few weeks back?

Reply:
@msmcconnell Yes, according to [this link](https://discourse.ros.org/t/new-gpg-keys-deployed-for-packages-ros-org/9454/9), "The new public key uses the same private key as before, so that fingerprint is still correct." 

I believe they issued a new cert (extending the expiration date) but the key pair remained the same. I verified in the build docker image that it's got the new cert (expiring in 2025):
```
pub   rsa4096 2019-05-30 [SC] [expires: 2025-06-01]
      C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
uid           [ unknown] Open Robotics <info@osrfoundation.org>
```